src/styles/_catalog.scss

```diff
@@ -14,9 +19,30 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    box-sizing: border-box;
   }
+}
+
+@media (min-width: $breakpoint-phone) {
```

Comment:
we have two variants how to use `@media`. Either selector inside media or media inside selector. 
But usually we use second variant
Example:
```
.selector {
  @media (...) {
    ...
  }
  
  @media (...) {
    ...
  }
 
  ....
}
```